examples/tools/android_native_run/Options.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Options.h"
+
+
+#include <fstream>
+#include <sstream>
+#include <getopt.h>
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+namespace
+{
+  string GetClassPath(string const &optarg) {
+  ifstream dependencies(optarg);
+  string line;
+  stringstream Paths;
+  while (getline(dependencies, line)) {
+    Paths << line << ":";
+  }
+  return Paths.str();
+  }
+}
+
+namespace
+{
+  vector<string> other_libraries(string const &optarg) {
+  ifstream Other_libraries(optarg);
+  string line;
+  vector<string> tab_other_libraries;
+  while (getline(Other_libraries, line))
+  {
+    tab_other_libraries.emplace_back(line);
+  }
+
+  return tab_other_libraries;
+  }
+}
+
+static int verbose_flag;
+
+javaOption options(int argc, char **argv, const char *shortopts,
+                   const struct long_options *longopts, int *indexptr) {
+  int option;
+  javaOption opt;
+
+  while (1) {
+    static struct option long_options[] = {
+        {"dependencies", required_argument, 0, 'd'},
+        {"library", required_argument, 0, 'l'},
```

Comment:
```suggestion
        {"libraries", required_argument, 0, 'l'},
```
I don't remember `getopt`'s documentation, but if there is a way to specify a help message it would be nice, if not it's alright.

Reply:
Done